.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
- [ ] Please as requested by the previous reviewer, Kindly make sure your Pull request has a descriptive summary describing the changes that you introduced to the branch. This will make your work look more professional. Kindly check [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects) for more details on a professional pull request.

- [Optional] Please always use descriptive commit messages - each commit should be small enough to describe it in one sentence. The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). This will make your code more professional. A good example of a descriptive commit message is Add markup for index.html and a bad example is added file.